### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-270.6] Filter field: user cannot enter value with keyboard.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11293]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
NGX Platform Table, If Column is filterable then on click of Filter icon in header of column we get Filter option text box. But The user is not able to access the 'Filter' edit field within the popup of the table header to enter any value. It must be possible to interact/activate each & every interactive UI, using the keyboard. 

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
NGX Platform Table toolbar menu filter option
NGX Platform Table, make Column filterable. on click of Filter icon in header of column we get Filter option text box. But The user is not able to access the 'Filter' edit field within the popup of the table header to enter any value. 
#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes
#### Is there anything else we should know?
Attaching screenshot for reference
<img width="1246" alt="Screenshot 2024-01-21 at 6 18 19 PM" src="https://github.com/SAP/fundamental-ngx/assets/157268666/1114cad7-827e-43a3-b07f-6dc42cb1d3fb">

**Comment by droshev** [comment]
@JyotiKumari1507 I was not able to reproduce the issue:
![table-filter-keyboard](https://github.com/SAP/fundamental-ngx/assets/4380815/0b70042a-aabc-4aca-b563-69cc7067aa49)
